pkg/snapstore/gcs_snapstore.go

```diff
@@ -83,21 +98,43 @@ func NewGCSSnapStore(config *brtypes.SnapstoreConfig) (*GCSSnapStore, error) {
 	}
 	gcsClient := stiface.AdaptClient(cli)
 
-	return NewGCSSnapStoreFromClient(config.Container, config.Prefix, config.TempDir, config.MaxParallelChunkUploads, config.MinChunkSize, gcsClient), nil
+	return NewGCSSnapStoreFromClient(config.Container, config.Prefix, config.TempDir, config.MaxParallelChunkUploads, config.MinChunkSize, chunkDirSuffix, gcsClient), nil
 }
 
 // NewGCSSnapStoreFromClient create new GCSSnapStore from shared configuration with specified bucket.
-func NewGCSSnapStoreFromClient(bucket, prefix, tempDir string, maxParallelChunkUploads uint, minChunkSize int64, cli stiface.Client) *GCSSnapStore {
+func NewGCSSnapStoreFromClient(bucket, prefix, tempDir string, maxParallelChunkUploads uint, minChunkSize int64, chunkDirSuffix string, cli stiface.Client) *GCSSnapStore {
 	return &GCSSnapStore{
 		prefix:                  prefix,
 		client:                  cli,
 		bucket:                  bucket,
 		maxParallelChunkUploads: maxParallelChunkUploads,
 		minChunkSize:            minChunkSize,
 		tempDir:                 tempDir,
+		chunkDirSuffix:          chunkDirSuffix,
 	}
 }
 
+// isEmulatorEnabled checks if the fake GCS emulator is enabled
+func isEmulatorEnabled() bool {
+	isFakeGCSEnabled, ok := os.LookupEnv(envEmulatorEnabled)
+	if !ok {
+		return false
+	}
+	emulatorEnabled, err := strconv.ParseBool(isFakeGCSEnabled)
+	if err != nil {
+		return false
+	}
+	return emulatorEnabled
+}
+
+// ConfigureGCSEmulator configures the fake GCS emulator
+func ConfigureGCSEmulator(chunkDirSuffix *string, opts []option.ClientOption, emulatorConfig gcsEmulatorConfig) {
+	endpoint := emulatorConfig.endpoint
+	os.Setenv("STORAGE_EMULATOR_HOST", strings.TrimPrefix(endpoint, "http://"))
```

Comment:
```suggestion
	err := os.Setenv("STORAGE_EMULATOR_HOST", strings.TrimPrefix(emulatorConfig.endpoint, "http://"))
```
You can avoid creating an unnecessary variable here. Also, please handle the error returned.